ui/src/apps/searchLogs/components/utils.ts

```diff
@@ -109,33 +118,46 @@ export function parseSearchingParams(resp: LogsearchTaskGroupResponse): Params {
   }
 }
 
-function getComponents(tasks: LogsearchTaskModel[]): LogsearchSearchTarget[] {
-  const targets: LogsearchSearchTarget[] = []
+export function parseHistoryStatsParams(
+  resp: LogsearchTaskGroupModel
+): StatsParams {
+  const { start_time, end_time, min_level, patterns } =
+    resp?.search_request || {}
+  const startTime = start_time ? moment(start_time) : null
+  const endTime = end_time ? moment(end_time) : null
+  return {
+    timeRange: [startTime, endTime] as RangeValue<moment.Moment>,
+    logLevel: min_level ?? 0,
+    searchValue: patterns && patterns.length > 0 ? patterns.join(' ') : '',
+    stats: resp?.target_stats,
+  }
+}
+
+function getComponents(tasks: LogsearchTaskModel[]): UtilsRequestTargetNode[] {
+  const targets: UtilsRequestTargetNode[] = []
   tasks.forEach((task) => {
-    if (task.search_target === undefined) {
+    if (task.target === undefined) {
       return
     }
-    targets.push(task.search_target)
+    targets.push(task.target)
   })
   return targets
 }
 
 export function getGRPCAddress(
```

Comment:
This function is no longer useful I guess.

Reply:
Yes, just remove this funtion and use `target.display_name`